src/content/docs/backend/API/unit-group_sets.md

```diff
@@ -0,0 +1,360 @@
+| group_id     | id of the group      | path           | integer   | Yes       |
+
+- Response: `200 OK`
+
+- Example Request:
+  ```bash
+   curl -X GET --header 'Accept: application/json' --header 'Username: aadmin' --header 'Auth_Token: Say13yA497H8wSFFXZxh' 'http://localhost:3000/api/units/1/group_sets/2/groups/3/members'
+  ```
+- Response Body:
+  ```json
+  [
+    {
+      "student_username": "TESTREQ",
+      "id": 4,
+      "student_first_name": "TEST",
+      "student_last_name": "REQ",
+      "student_nickname": "TEST1",
+      "target_grade": "A"
+    },
+    {
+      "student_username": "TEST2REQ2",
+      "id": 5,
+      "student_first_name": "TEST2",
+      "student_last_name": "REQ2",
+      "student_nickname": "TEST2",
+      "target_grade": "B"
+    }
+  ]
+  ```
+---
+### DELETE: Delete a group
+
+DELETE /api/units/{unit_id}/group_sets/{group_set_id}/groups/{group_id}
+
+- URL: `/api/units/{unit_id}/group_sets/{group_set_id}/groups/{group_id}`
+- Method: `DELETE`
+- Parameter:
+
+| Parameter    | Description          | Parameter Type | Data Type | Mandatory |
+| ------------ | -------------------- | -------------- | --------- | --------- |
+| Username     | Username             | header         | string    | Yes       |
+| Auth_Token   | Authentication token | header         | string    | Yes       |
+| unit_id      | The unit id          | path           | integer   | Yes       |
+| group_set_id | Id of the group set  | path           | integer   | Yes       |
+| group_id     | id of the group      | path           | integer   | Yes       |
+
+- Response: `200 OK`
+
+- Example Request:
+
+  ```bash
+   curl -X DELETE --header 'Accept: application/json' --header 'Username: aadmin' --header 'Auth_Token: Say13yA497H8wSFFXZxh' 'http://localhost:3000/api/units/1/group_sets/2/groups/3'
+  ```
+- Response Body:
+  ```
+  True
+  ```
+---
+### PUT: Update a group
+
+PUT /api/units/{unit_id}/group_sets/{group_set_id}/groups/{group_id}
+
+- URL: `/api/units/{unit_id}/group_sets/{group_set_id}/groups/{group_id}`
+- Method: `PUT`
+- Parameter:
+
+| Parameter           | Description                            | Parameter Type | Data Type | Mandatory |
+| ------------------- | -------------------------------------- | -------------- | --------- | --------- |
+| Username            | Username                               | header         | string    | Yes       |
+| Auth_Token          | Authentication token                   | header         | string    | Yes       |
+| unit_id             | The unit id                            | path           | integer   | Yes       |
+| group_set_id        | Id of the group set                    | path           | integer   | Yes       |
+| group_id            | id of the group                        | path           | integer   | Yes       |
+| name                | name of the group                      | body           | string    | No        |
```

Comment:
Maybe it would be best to include that the name, tutorial_id, capacity_adjustment, and locked parameters are included in an overall 'group' hash

Reply:
Hi Jake, I have updated the Parameters as per agreed format in the stand-up meeting.